test/deny_SUITE.erl

```diff
@@ -134,14 +118,7 @@ test_bad_deny(_Config) ->
     Options = #{},
 
     %% Deny requires a target and a target - leave off target and it should fail
-    Json = <<"{ \"action\": \"deny\",
-               \"actuator\": {
-                  \"type\": \"network-scanner\",
-                  \"specifiers\": \"scanner01\"
-                           },
-               \"modifiers\": { \"response\": \"ack\",
-                              \"where\": \"perimeter\"}
-             }">>,
+    Json = ?JSON_DENY_WO_TARGET,
```

Comment:
The variable "JSON_DENY_WO_TARGET" on line 121 does not respect the format defined by the regular expression '"^([A-Z][0-9a-zA-Z]*)$"'.